apps/session-manager/src/types/token.ts

```diff
@@ -36,3 +36,13 @@ export interface IFIMSTokenTag {
 }
 export const FIMSToken = tag<IFIMSTokenTag>()(t.string);
 export type FIMSToken = t.TypeOf<typeof FIMSToken>;
+
+export interface UserTokens {
```

Comment:
we should prefer type instead of interfaces

Reply:
done in https://github.com/pagopa/io-auth-n-identity-domain/pull/22/commits/0aac1a7af5e32060657c60e53ef458e41e4e445c